google/cloud/spanner_v1/streamed.py

```diff
@@ -147,6 +148,17 @@ def __iter__(self):
             except StopIteration:
                 return
 
+    def to_dataframe(self):
+        """Returns the response in a pandas dataframe of the StreamedResultSet object
```

Comment:
Let's indicate "all rows" to be clear that this will download all data at once.

```suggestion
        """Creates a pandas DataFrame of all rows in the result set
```